project/src/com/company/entities/Savings.java

```diff
@@ -0,0 +1,43 @@
+package com.company.entities;
+
+public class Savings {
+    Account account;
+    double balance;
+
+    //constructor
+    public Savings(Account account) {
+        this.account = account;
+        this.balance = 0;
```

Comment:
redundant initialization, default value for a double is 0.0 if left uninitialized

Reply:
Fixed.